src/main/java/it/pagopa/pn/commons/configs/IsMVPParameterConsumer.java

```diff
@@ -1,37 +1,45 @@
 package it.pagopa.pn.commons.configs;
 
-import it.pagopa.pn.commons.abstractions.impl.AbstractCachedSsmParameterConsumer;
+import it.pagopa.pn.commons.abstractions.ParameterConsumer;
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Value;
 
 import java.util.Optional;
 
+@Slf4j
 public class IsMVPParameterConsumer {
```

Comment:
Potrebbe essere rinominata la classe in MVPParameterConsumer? Solitamenete i verbi vengono utilizzati per i metodi (così si eviterebbe anche il "isMVPParameterConsumer.isMVP). Poi se un giorno il paremetro dell'MVP non è più un booleno, si eviterebbe anche di cambiare il nome della classe.

Reply:
yes sono d'accordo